### DISCUSSION TITLE
Consider partial releases

### DISCUSSION THREAD

**Discussion by vvranceanu** [chromiecraft/chromiecraft#6859] [Suggestions]
Consider releasing future raids in steps, aka batches of bosses. Leaving a bit the open-source way of making things aside(cause i'm sure some people have some focused work when release dates are coming), this should make it in theory that work is more focused for testing and development and the effort for a release wouldn't be as big(i can imagine when issues start pilling up, morale can suffer. this is much more pronounced when having bug reports for 6 bosses vs 2 bosses).

I think at least BT/SWP will be some big raids with considerable amounts of work going into them, i can't see how they are comparable to the smaller ones and SWP was afaik also released staggered by blizzard over a couple of weeks.

I see this as a win-win for the consumers as well as the devs, since people get to enjoy content potentially faster and devs have a smaller amount of checkboxes for a release. I also can see people having an easier time going over a boss that wasn't quite figured out yet by simply releasing another instead at the time or leaving it out(if 2-3-4 bosses was planned for a step)

**Comment by balleny** [comment]
I am still throwing in my suggestion i bring up on a regular basis:

release the raids early on with all loot removed.. People can still get achievements, there will be a decent amount of feedback because the barrier to enter is lower compared to PTR and feedback for tuning can also be collected (and practically, there is no harm in it being potentially overtuned in one week or undertuned in the second week, there is no harm if a boss cannot be passed
(it would make sense to e.g. remove identical trash packs so that each type or combination of trash pack is present only once)
(and then e.g. release it 2 weeks after) - practically a final flip/testphase on the live realm.

Gated releases are a thing and were used by blizz on the original version but not used in the classic release.

**Reply by vvranceanu** [reply]
I mentioned the staggered release as an example of a behemoth company that used it to cater to paying customers and got away with it, it's a proven working delivery method. It shifts from a V cycle to more of an agile(feature by feature) delivery(again, the open source argument idk to which extend should be had, since we're talking about release dates for specific content) which is more focused and more humane if you ask me. I don't think this being blizzlike or not is worth discussing, the raid releases are currently are not blizzlike.

I don't see releasing scaffolded content working. It's ptr without the gm commands. With no loot, many will have no incentive to go. Why would i waste hours of 20-25 people's time for something i could have tested on the ptr in 5 minutes with 2 gm commands. It also leaves way to crashing-the-server type of bugs to go on live. It will weird out newcomers, it will break immersion for people doing it. It will be complicated to manage releases: when does the scaffolding go off? Do we keep the scaffolding on if its not ready or not? etc. If a player wasn't interested in going to the ptr and testing, will he even take testing on live seriously? How do we manage changes? PTR is updated twice a day, live is updated once a week. There will be hotfixes, do we restart live every time we apply one? Or do we wait the whole week to pass?

**Comment by Nyeriah** [comment]
There is a certain feeling of disappointment and frustration when the raid night ends a few or many bosses short when you're supposed to be clearing the entire instance. We understand that there's also some frustration when raids don't work as they should or they take too long to be released. Still, after pondering we still believe releasing everything in one go, after addressing as many issues as possible, gives the better player experience out of the two options.

Therefore, thanks for your suggestion, but this is a route we are not taking.
